backend/api/notifications/notifications.py

```diff
@@ -101,6 +108,30 @@ def __determine_message_recipients(is_global: bool = False,
 
         return []
 
+    @staticmethod
+    def send_email_for_notification(notification: NotificationMessage):
+
+        if not EMAIL['ENABLED']:
+            return
+
+        email_recipient = notification.user.email
+        from email.message import EmailMessage
+        import smtplib
+        msg = EmailMessage()
+        msg.set_content('You have received a new notification in TFRS.\nPlease sign in to view it.')
+        msg['Subject'] = 'TFRS Notification'
+        msg['From'] = EMAIL['FROM_ADDRESS']
+        msg['To'] = email_recipient
+
+        print(msg.as_string())
+
+        with smtplib.SMTP(host=EMAIL['SMTP_SERVER_HOST'],
+                          port=EMAIL['SMTP_SERVER_PORT']) as server:
+            server.send_message(msg)
+
+        print('sending notification to {}'.format(email_recipient))
```

Comment:
Is this just for us to keep track while the server is running?

Reply:
Yes, but it might be superfluous. I'll disable the print()